app/templates/components/layer-record-views/tax-lot.hbs

```diff
@@ -210,6 +210,11 @@
       </ul>
     </div>
   </div>
+  {{#if (eq this.model.notes 1)}}
```

Comment:
~Can we do `{{#if this.model.notes}}`~

Nevermind, I see that the numeric from EDM's change to PLUTO corresponds to a specific piece of text in the template. The idea is that there could be more notes, and you might want to show them all.

One problem I'm seeing is that there is duplicate text spread across three files. I'd really like to see a single component used for these callouts.

Reply:
Right. The `notes` field _could_ be a comma separated list — e.g. `1,2,4`. 

Long-term solution, would be a notes component that's passed an array and loops thru them to output multiple `div.callout` elements. Each actual note content will ultimately be provided in a separate table, so there'd need to be a query for those. Tho for now, I think it's okay the the note content is hardcoded, but agree that it should only be hardcoded once. 

On the other hand, this exact situation being solved for is very much an edge case, has not happened before, and will likely not happen again. So maybe it's fine we don't overthink it… LGTM.

Reply:
Yes that is one way to do it. If you need help doing it let me know or bounce this over to me please.

Reply:
The text on the tax lot view is slightly different actually. Ideally we will be undoing this work in a few months once the lawsuit is resolved. If future note demands come up, I think we should deal with componentizing and improving it then, but I'll defer to @allthesignals if he wants to implement it differently.